vitrina/datasets/models.py

```diff
@@ -109,7 +108,7 @@ class Dataset(models.Model):
     financing_priorities = models.TextField(blank=True, null=True)
     financing_received = models.BigIntegerField(blank=True, null=True)
     financing_required = models.BigIntegerField(blank=True, null=True)
-    will_be_financed = models.BooleanField()
+    will_be_financed = models.BooleanField(blank=True, null=True)
```

Comment:
```suggestion
    will_be_financed = models.BooleanField(blank=True)
```

In which case would we need a NULL value? I would suggest adding `null=True` only if we really need it, in most cases types who have an empty or falsy value does not need `null=True`.